CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -14,6 +14,14 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ### Package `clientv3`
 - Reverted the fix to [auth invalid token and old revision errors in watch](https://github.com/etcd-io/etcd/pull/14995).
 
+### Security
+- Use [distroless base image](https://github.com/etcd-io/etcd/pull/15016) to address critical Vulnerabilities.
+- Bumped [some dependencies](https://github.com/etcd-io/etcd/pull/15018) to address some HIGH Vulnerabilities.
```

Comment:
Could you please, link explicitly to: https://pkg.go.dev/vuln/GO-2022-1144

Reply:
There are lots of CVEs, all of which are included in the related PR. such as 
- https://github.com/etcd-io/etcd/pull/15016
- https://github.com/etcd-io/etcd/pull/15018